.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "go.buildTags": "all,smoke",
+    "go.buildTags": "all",
```

Comment:
Oh this makes sense, since the tests are `!smoke`. Whoops.

Reply:
I just got fed up of having to change this all the time to be able to run tests in vscode, I don't think it will have affected CI in any way.